packages/golden-layout/src/controls/Tab.ts

```diff
@@ -181,25 +176,16 @@ export default class Tab {
 
   /**
    * Callback when the contentItem is focused in
-   *
-   * Why [0].focus():
-   * https://github.com/jquery/jquery/commit/fe5f04de8fde9c69ed48283b99280aa6df3795c7
-   * From jquery source: "If this is an inner synthetic event for an event with a bubbling surrogate (focus or blur),
-   * assume that the surrogate already propagated from triggering the native event and prevent that from happening
-   * again here. This technically gets the ordering wrong w.r.t. to `.trigger()` (in which the bubbling surrogate
-   * propagates *after* the non-bubbling base), but that seems less bad than duplication."
    */
   _onTabContentFocusIn() {
-    if (
-      isComponent(this.contentItem) &&
-      !this.contentItem.container._contentElement[0].contains(
-        document.activeElement
-      )
-    ) {
-      // jquery 3.4.0 and later, jquery method optimizes out the focus from
-      // happening in proper order. Can use HTMLElement.focus() to avoid.
-      this.contentItem.container._contentElement[0].focus(); // [0] needed to use dom focus, not jquery method
-    }
+    // Ensure only one tab is marked as having focus at a time.
+    // In Firefox, if the focused element is removed from the DOM,
+    // the focusout event won't trigger. This can result in two tabs
+    // being erroneously marked as focused if the user's DOM element
+    // is removed and they click another tab. To prevent this, we
+    // remove existing "lm_focusin" classes first.
+    $('lm_focusin').removeClass('lm_focusin');
```

Comment:
Can you scope this to within the layout manager root? Also shouldn't it be `.lm_focusin`? You sure we need this?
```suggestion
    $('.lm_focusin', this._layoutManager.root).removeClass('lm_focusin');
```

Reply:
`root.element`, yeah. 

You can replicate the issue in firefox only by clicking the table sidebar menu button on a table, then clicking inside another table, and both will show as selected.

![highlight](https://user-images.githubusercontent.com/1576283/236322666-142961df-1fac-432d-9c85-80705c6f2f52.gif)